services/conservation_portfolios/src/routes/targets.ts

```diff
@@ -0,0 +1,46 @@
+import { Router } from 'restify-router';
+import { EHFunction } from '../util/types_conf';
+import { PortfoliosByTarget } from '../util/types_data';
+
+interface TargetsService {
+  getPortfoliosByTarget: () => Promise<Array<PortfoliosByTarget>>;
+}
+
+export default (errorHandler: EHFunction, TargetsService: TargetsService) => {
+  const router = new Router();
+
+  /**
+   * @apiGroup s_portfolios_ca
+   * @api {get} /portfolios-ca/target PortfoliosByTarget
+   * @apiName PortfoliosByTarget
+   * @apiVersion 1.0.0
+   * @apiDescription
+   * Values of all portfolios for a specific target within a given area
+   *
+   * @apiParam (Query params) {String} areaType area type
+   * @apiParam (Query params) {String|Number} areaId area id
+   * @apiParam (Query params) {Number} targetId target id to filter results
+   *
+   * @apiSuccess {Object} result
```

Comment:
The result is an array of objects and you're missing the properties description

Reply:
Now returns an object and properties description was included